content/w/oskar-aleksander.md

```diff
@@ -34,11 +34,15 @@ Oskar made his debut as one of the contenders in Johnny Blade's Insane Wheel of
 
 #### Becoming Oskar Aleksander
 
-After his victory over [Marcelito](@/w/marcelito.md), [Sambor](@/w/sambor.md) and [Aron Wake](@/w/aron-wake.md) at [Co Za Noc](@/e/ppw/2024-10-26-ppw-co-za-noc.md), in a post-match promo he declared that he earned the right to decide what his ringname would be.
-Since then he would be billed as "Oskar Aleksander". At the same show, in a backstage interview, he became affiliated with an unnamed female "[Talent Agent](@/w/agentka-agatka.md)", who became his manager.
+After his victory over [Marcelito](@/w/marcelito.md), [Sambor](@/w/sambor.md) and [Aron Wake](@/w/aron-wake.md) at [Co Za Noc](@/e/ppw/2024-10-26-ppw-co-za-noc.md), in a post-match promo he declared that he earned the right to decide what his ringname would be, and he also aligned himself with [Agentka Agatka](@/w/agentka-agatka.md). Since then he's been billed as "Oskar Aleksander". At the same show, in a backstage interview, he became affiliated with an unnamed female "[Talent Agent](@/w/agentka-agatka.md)", who became his manager.
```

Comment:
```suggestion
After his victory over [Marcelito](@/w/marcelito.md), [Sambor](@/w/sambor.md) and [Aron Wake](@/w/aron-wake.md) at [Co Za Noc](@/e/ppw/2024-10-26-ppw-co-za-noc.md), in a post-match promo he declared that he earned the right to decide what his ringname would be. Since then he's been billed as "Oskar Aleksander". At the same show, in a backstage interview, he became affiliated with an unnamed female "[Talent Agent](@/w/agentka-agatka.md)", who became his manager.
```